pytest-selenium/tests/test_attribution_pom.py

```diff
@@ -0,0 +1,138 @@
+
+from time import sleep
+
+import pytest
+from selenium import webdriver
+
+from pages.content import Content
+from tests import markers
+
+
+class TestCitationtexturl():
+
+    # c476302 citation text shows url for current page
+    @markers.test_case("C476302")
+    @markers.parametrize("book_slug,page_slug", [("college-physics", "preface")])
+    @markers.nondestructive
+
+    def test_citation_url_matches_page_url(self, selenium, base_url, book_slug, page_slug):
+
+    #Given - the page {base_url}/books/college-physics/pages/preface
+    #When - the page loads
+        content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+        url_before_attribution_click = content.current_url
+        #print(content.current_url)
+        
+    #AND the attribution section is expanded
+        attribution = content.attribution
+        sleep(1)
+        attribution.attribution_click()
+        url_after_attribution_click = content.current_url
+        #print(content.current_url)
+     
+    #Then - the urls in the the attribution should reference current page in rex
+        assert url_before_attribution_click == url_after_attribution_click
```

Comment:
Awesome!

Reply:
Agreed

Reply:
the test assertion `the urls in the the attribution should reference current page in rex` is about making sure the urls in the attribution text reference the current page, i don't see that happening here?

Reply:
The scenario is making sure page url doesn't change when we expand the attribution on a page:
I am storing the page url in `url_before_attribution_click'
Then click on Attribution and storing the new url in `url_after_attribution_click`
comparing the 2 url's in assert statement.

Did you mean this or am I not getting your question.